pdata/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/json-iterator/go v1.1.12
 	github.com/stretchr/testify v1.8.0
+	go.uber.org/zap v1.21.0
```

Comment:
Where is this coming from?

Reply:
Previously wished to use `zap` to have a debug log for unknown field. Now removed it.